toolkit/recordreplay/ProcessRecordReplay.cpp

```diff
@@ -317,9 +319,9 @@ MOZ_EXPORT void RecordReplayInterface_Initialize(int* aArgc, char*** aArgv) {
     // existed and won't capture it in the recording itself, which
     // is ideal for security.
 #ifdef XP_WIN
```

Comment:
It seems like we'd need to duplicate this code rather than changing it, otherwise `RECORD_REPLAY_API_KEY` will still exist in this environment _also_?

Reply:
Good point. Out of curiosity:
- how can these variables be captured in the recording?
- what is this code actually removing? It's not removing the process' environment variables, otherwise we wouldn't have had the authentication issue in the first place. Is it working on a copy of these variables?

Reply:
Fixed in #640